src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -931,3 +824,185 @@ def update_agents(cmd, client, resource_group_name, cluster_name, https_proxy=""
         raise CLIError(str.format(consts.Update_Agent_Failure, error_helm_upgrade.decode("ascii")))
 
     return str.format(consts.Update_Agent_Success, connected_cluster.name)
+
+
+def upgrade_agents(cmd, client, resource_group_name, cluster_name, kube_config=None, kube_context=None,
+                   no_wait=False, arc_agent_version=None):
```

Comment:
i dont see no_wait parameter being used in this command.

Reply:
no_wait is also not being used in update_agents. I didnt as it was not there. Also, --atomic uses --wait.

Reply:
Let's remove this param if its not being used.